### DISCUSSION TITLE
考虑加个特性:  Input 组件支持 trim prop , 自动过滤掉两边空格?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#44712] [Ideas]
```jsx
 <Form.Item name="app_id" label="app_id">
              <Input allowClear trim />
            </Form.Item>
```

**Comment by Yuiai01** [comment]
这个在 `onChange` 的时候自行处理一下是不是就可以了？

**Reply by vaynevayne** [reply]
你不觉得很麻烦啊, 不如 vue2 的 指令

**Reply by vaynevayne** [reply]
而且不是说 formItem 下的组件不要动 value onchange 嘛

**Reply by afc163** [reply]
normalize 即可

**Reply by Yuiai01** [reply]
<img width="1008" alt="image" src="https://github.com/ant-design/ant-design/assets/112228030/5ee3a5d7-0f2a-41dc-918d-2ce55b0cbc6b">
应该是这个

**Reply by q1uxu** [reply]
> normalize 即可

能否让 ConfigProvider 组件支持给 Input 配置 normalize 属性呢？一个个给Input组件配置normalize太麻烦了，HOC包裹二次封装Input组件是可以不用一个个配置，但是要从不同的路径导入组件。
